src/main/java/com/conveyal/datatools/manager/jobs/MonitorServerStatusJob.java

```diff
@@ -248,14 +258,37 @@ private void waitAndCheckInstanceHealth(String waitingFor) throws InstanceHealth
         checkInstanceHealth();
     }
 
+    /**
+     * Helper method to initiate to first attempt at checking the instance health.
+     */
+    private void checkInstanceHealth() throws InstanceHealthException {
+        checkInstanceHealth(1);
+    }
+
     /**
      * Checks whether the instance is running. If it has entered a state where it is stopped, terminated or about to be
      * stopped or terminated, then this method throws an exception.
```

Comment:
javadoc should mention behavior when attemptNumber exceeds max retries. Also, might be useful to mention that the method should be initiated with the no-arg `checkInstanceHealth` method.